internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -103,12 +103,12 @@ func (v *vmConsoleProxy) Reconcile(request *common.Request) ([]common.ReconcileR
 		return []common.ReconcileResult{}, nil
 	}
 
-	reconcileFunc = append(reconcileFunc, reconcileServiceAccountsFuncs(*v.serviceAccount.DeepCopy()))
-	reconcileFunc = append(reconcileFunc, reconcileClusterRoleFuncs(*v.clusterRole.DeepCopy()))
-	reconcileFunc = append(reconcileFunc, reconcileClusterRoleBindingFuncs(*v.clusterRoleBinding.DeepCopy()))
-	reconcileFunc = append(reconcileFunc, reconcileConfigMapFuncs(*v.configMap.DeepCopy()))
-	reconcileFunc = append(reconcileFunc, reconcileServiceFuncs(*v.service.DeepCopy()))
-	reconcileFunc = append(reconcileFunc, reconcileDeploymentFuncs(*v.deployment.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileServiceAccounts(*v.serviceAccount.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileClusterRole(*v.clusterRole.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileClusterRoleBinding(*v.clusterRoleBinding.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileConfigMap(*v.configMap.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileService(*v.service.DeepCopy()))
+	reconcileFunc = append(reconcileFunc, reconcileDeployment(*v.deployment.DeepCopy()))
```

Comment:
Can you inline the `reconcileFunc` slice into the `common.CollectResourceStatus()` function?
It will look like this:
```go
reconcileBundleResults, err := common.CollectResourceStatus(request,
	reconcileServiceAccountsFuncs(*v.serviceAccount.DeepCopy()),
	reconcileClusterRoleFuncs(*v.clusterRole.DeepCopy()),
	reconcileClusterRoleBindingFuncs(*v.clusterRoleBinding.DeepCopy()),
	// etc ...
)
```

Can you also directly `return` the result of the `common.CollectResourceStatus()` function?

And similarly in the `Cleanup()` function, it's not needed to create an `objects` slice.

Reply:
Yep, done. Thanks!